lib/src/fnc/type.rs

```diff
@@ -115,6 +115,10 @@ pub fn thing((arg1, arg2): (Value, Option<Value>)) -> Result<Value, Error> {
 	} else {
 		match arg1 {
 			Value::Thing(v) => v.into(),
+			Value::Strand(v) => match Thing::try_from(v) {
+				Ok(v) => v.into(),
+				_ => Value::None,
```

Comment:
We should return an error here.

Reply:
Should we then also return an error if something else than a Thing or a Strand is passed there? I went for the Value::None route because that was essentially already happening